packages/phone-number-privacy/signer/src/domain/quota/domainQuota.interface.ts

```diff
@@ -0,0 +1,12 @@
+import { KnownDomain } from '@celo/identity/lib/odis/domains'
+import { Transaction } from 'knex'
+import { DomainState } from '../../database/models/domainState'
+
+export interface IDomainQuotaService {
+  doesHaveRemainingQuota: (domain: KnownDomain, domainState: DomainState) => Promise<boolean>
+  increaseQuotaCount: (
```

Comment:
`increaseQuotaCount` is a bit misleading to me because increasing quota might be read to imply the domains quota is being increased (i.e. such that more requests can be made, rather than fewer.) I might suggest something like `chargeQuota`
```suggestion
  chargeQuota: (
```